decidim-core/config/locales/es.yml

```diff
@@ -7,6 +7,22 @@ es:
         welcome: Bienvenido/da %{organization}!
     menu:
       home: Inicio
+  devise:
+    mailer:
+      invitation_instructions:
+        accept: Aceptar invitación
+        accept_until: Ésta invitación caducará el día %{due_date}.
+        hello: Hola %{email}
+        ignore: |-
+          Si no queires aceptar la invitación ignore este correo.<br />
```

Comment:
`s/queires/quieres` (sounds Galician LOL)
`s/ignore/ignora` in order to be consisten with the person number form.